javascript-and-json/main.js

```diff
@@ -0,0 +1,23 @@
+var array = [ {
+  "isbn": "1235846123",
+  "title": "Hello World",
+  "author": "Standard Code",
+},
+{
+  "isbn": "987812315",
+  "title": "Goodbye World",
+  "author": "Reverse Order",
+},
+{
+  "isbn": "159753654852",
+  "title": "Dlrow Eyboog",
+  "author": "Redro Dradnats",
+},
+];
```

Comment:
You're missing some console logs for this array, make sure you're following all the directions. You should be logging out the value of this array, along with the `typeof`